sos/cleaner/__init__.py

```diff
@@ -129,6 +129,16 @@ def validate_map_file(self):
                 self.log_error(
                     "ERROR: map file %s does not exist, will not load any "
                     "obfuscation matches" % self.opts.map_file)
+        else:
+            with open(self.opts.map_file, 'r') as mf:
+                try:
+                    json.load(mf)
```

Comment:
It is pitty we load+parse the `json` file repeatedly (here just for a check and to throw away its content, and in https://github.com/sosreport/sos/blob/master/sos/cleaner/parsers/__init__.py#L60 to really store the `json` for each and every parser individually). If the file becomes huge, this costs us a bit (still irelevant low time compared to the execution time of cleaner).